dev.boxi.Boxi.yml

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env -S flatpak-builder --user --install --force-clean tmp
+
+app-id: dev.boxi.Boxi
+runtime: org.gnome.Platform
+runtime-version: '42'
+sdk: org.gnome.Sdk
+
+modules:
+  - name: vte
+    sources:
+      - type: git
+        url: https://gitlab.gnome.org/GNOME/vte
+        tag: 0.68.0
+        commit: 0f438924f9f8a858b1b82434c876e31c2de180d4
+    buildsystem: meson
+
+  - name: python3-flit_core
+    sources:
+      - type: file
+        url: https://files.pythonhosted.org/packages/ad/38/80992c4cb490a05474c886850ec818276d09c0a6722bd5b43bc487818ac7/flit_core-3.7.1-py3-none-any.whl
+        sha256: e454fdbf68c7036e1c7435ec7479383f9d9a1650ca5b304feb184eba1efcdcef
+    buildsystem: simple
+    build-commands:
+      - pip3 install --no-build-isolation --prefix=/app flit_core-3.7.1-py3-none-any.whl
+
+  - name: boxi
+    sources:
+      - type: git
+        url: https://github.com/allisonkarlitskaya/boxi
+        tag: 3
```

Comment:
I bet that yaml is throwing a warning. It should be `tag: '3'` since it expect a string.